### PR TITLE
Jsonintegration

### PR DESCRIPTION
**Pull request by EricSommerhalder**

### CODE REVIEW

**Comment by EricSommerhalder** [comment]
f the conflicts

File: index.html [hunk 1]
```diff
@@ -1,16 +1,5574 @@
...
-    <script>
-        Elm.Main.init({ node: document.getElementById("app") });
-    </script>
-</body>
\ No newline at end of file
+
+<pre id="elm"></pre>
+
+<script>
+try {
```
Comment (mdelez) [inline index.html]: why is the whole `app.js` here? 😅

File: src/Dropdown.elm [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+            , Attr.attribute "aria-orientation" "vertical"
+            , ariaLabelledby "menu-button"
+            , tabindex -1
+            ] 
+            [ div
+                [ class "py-1"
+                , role "none"
+                ]
+                (List.indexedMap (viewOption model) model.input.options)
+                {-[ a
```
Comment (mdelez) [inline src/Dropdown.elm:112]: is this commented code needed?

File: test_data.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+data = {"dropdown": {
+    "header" : "Best MTG color",
```
Comment (mdelez) [inline test_data.json:2]: might be better to use the word "label" here

File: with_json.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+</head>
+
+<body>
+    <div id="app"></div>
+    <script src="dropdown.js"></script>
+    <script type="text/javascript" src="test_data.json"></script>
+    <script>
+        for (var i in data){
+            if (i == "dropdown"){
+                console.log(data[i])
```
Comment (mdelez) [inline with_json.html:17]: please remove the console.log :)

**Review by mdelez — CHANGES_REQUESTED** [review]

**Pull request closed without merging**